plugins/webhooks/index.js

```diff
@@ -264,7 +299,7 @@ async function triggeredPipelines(pipelineFactory, scmConfig, branch, type, acti
     );
 
     pipelinesOnOtherBranch = pipelinesOnOtherBranch.filter(p =>
-        hasTriggeredJob(p, determineStartFrom(action, type, branch, null))
+        hasTriggeredJob(p, determineStartFrom(action, type, branch, null, releaseName, tagName))
```

Comment:
Is the argument of `isReleaseOrTagFiltering` unnecessary?

Reply:
I add `isReleaseOrTagFiltering`.